apps/pl-default/pattern-lab/_patterns/02-molecules-demo/utchero/utchero.twig

```diff
@@ -0,0 +1,18 @@
+{#
+/**
+ * @file Cards!
+ *   Documenting @molecules/card/_utchero.twig
+ */
+#}
+
+{% embed '@demo/_demo-block.twig'
+  with {
+    demo: utchero_demo
+  }
+%}
+  {% block content %}
+    {% include '@molecules/utchero/_utchero.twig' with {
+      utchero: utchero_1
+    } %}
+  {% endblock content %}
+{% endembed %}
```

Comment:
@bmartinez287 and @UTCbridget, so would we also update the template file for the hero block to be like this, with an includes statement pointing to this twig file? With the variables being piped in from the hero block at the top? 

Just making sure I get the connection of the design system to the project.

Reply:
yes, I have gotten to that part yet. But once this is ready this would be the parent of all the other heroes. Very similar to how you made the utchero full the parent for some view modes.